test/validators/movies/list.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Joi = require('joi');
+
+const ListMoviesValidator = require('../../../lib/validators/movies/list');
+
+describe('list movies validator', () => {
+
+  describe('title', () => {
+
+    it('is a string', () => {
+      const payload = { title: 'The Dark Knight' };
+      const result = Joi.validate(payload, ListMoviesValidator);
+
+      expect(result.error).to.be.null;
+      expect(result.value.title).to.be.a('string');
+    });
+
+  });
+
+  describe('release_year', () => {
+
+    it('is allowed', () => {
+      const payload = {
+        release_year: 2002
+      };
```

Comment:
nit: make this a single line to be consistent with the above test, or vice versa